earth_enterprise/src/common/CacheSizeCalculations.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 The Open GEE Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "khTypes.h"
+#include "khRefCounter.h"
+#include "khMetaData.h"
+#include <notify.h>
+
+#ifndef _INCL_GUARD
```

Comment:
We should name our include guards something specific to the file, like CACHE_SIZE_CALCULATIONS_H.

Also, I prefer to have everything (including the other include statements) inside the include guard. It won't change anything except possibly make compilation a bit faster.